packages/@o3r/azure-tools/src/cli/comment-pr.ts

```diff
@@ -20,7 +20,7 @@ program
     .choices(['Replace', 'Add', 'Skip'])
     .default('Add')
   )
-  .option('-I, --threadIdentifier <threadIdentifier>', 'Thread identifier', undefined)
+  .requiredOption('-I, --threadIdentifier <threadIdentifier>', 'Thread identifier')
```

Comment:
`threadIdentifier` should not be mandatory in case of `mode="Add"` (the default case)

Reply:
I made it mandatory since it is passed as a parameter to the function `findThreadsByIdentifier` on line 65 and this parameter is of type `string`: 
https://github.com/AmadeusITGroup/otter/blob/f703e9002c44dfff328937c061e15fe6d174c7c5/packages/%40o3r/azure-tools/src/cli/comment-pr.ts#L65

Reply:
```typescript
 const threads = opts.threadIdentifier ? await prService.findThreadsByIdentifier(repositoryId, pullRequestId, opts.threadIdentifier) : [];